bot.py

```diff
@@ -152,9 +154,20 @@ def __init__(self):
             dict[str, Any],
         ] = {}  # name differentiation between internal _listeners attribute
         self.__version__ = "v5.1.0"
+        self.__commit__ = self.get_commit()
         self.session = None
         self.mongo_database = MongoDatabase(self)
 
+    def get_commit(self) -> str | None:
+        with subprocess.Popen(
+            ["git", "rev-parse", "--verify", "HEAD"],
```

Comment:
Do we really need the entire hash? Why not replace `--verify` with `--short`?

Reply:
Yeah `--short` should be fine. If 7 characters are not enough, Git will just spit out a larger length.